docs/admin/access.rst

```diff
@@ -79,46 +68,47 @@ Project access control
 .. note::
 
     By turning on access control, all users are prohibited from accessing anything
-    within a given project, unless you add the permissions for them to do just that.
+    within the given project, unless you add the permissions for them to do just that.
 
 .. include:: /snippets/not-hosted-libre.rst
 
 Limit user's access to individual projects by selecting a different access
-control variation on the :guilabel:`Access` tab in the :guilabel:`Settings` of
-each respective project.  This automatically creates several groups for the
+control type from the :guilabel:`Access` tab in the :guilabel:`Settings` of
```

Comment:
It could be, but this is the sleeker way (I think) also used elsewhere.
It isn't so much "on the tab" as it is "in the tab", but moreso "beyond the tab on the tab page"?
It was:
"selecting a different access control variation on the Access tab in the Settings of each respective project."

Reply:
ok, fine by me...